routes/api.php

```diff
@@ -49,3 +51,10 @@
         Route::delete('/{user}', DeleteUserController::class)
             ->whereNumber('user');
     });
+
+Route::post('employees', StoreEmployeesController::class)->name('employees');
+Route::get('employees', ListEmployeesController::class);
+
+Route::post('tasks', UpsertTaskController::class)->name('tasks');
+Route::get('tasks/{task}', FindTaskController::class);
+Route::get('tasks', ListTasksController::class);
```

Comment:
What about having sth like this?
```suggestion
Route::prefix('employees')->group(function(){
    Route::post('/', StoreEmployeesController::class);
    Route::get('/', ListEmployeesController::class);
});

Route::prefix('tasks')->group(function(){
    Route::post('/', UpsertTaskController::class);
    Route::get('', ListTasksController::class);
    Route::get('/{task}', FindTaskController::class);
});
```
It could be more readable no? what do u think

Reply:
Liked the idea, implemented it and added a name to the prefix and controllers to use the route helper